src/common/ManagedCommon/ManagedCommon.csproj

```diff
@@ -17,6 +17,8 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.Win32.Registry" Version="5.0.0" />
+    <PackageReference Include="System.Management" Version="6.0.0" />

```

Comment:
this requires installer update

Reply:
Check Product.wxs file. You need to ship Microsoft.Win32.Registry and  System.Management dlls with binaries that are using those dlls

Reply:
@stefansjfw I updated the Product.wxs file, can you confirm if I did it correctly?